source/core/parameters.cc

```diff
@@ -171,19 +171,13 @@ namespace Parameters
         method = TimeSteppingMethod::bdf2;
       else if (sv == "bdf3")
         method = TimeSteppingMethod::bdf3;
```

Comment:
Sdirk options are still in the declare_parameters explanatory text, they should be removed